src/libvtune.cc

```diff
@@ -12,9 +12,11 @@
 #include "libvtune.h"
 #include "vtuneapi.h"
 
+using namespace std;
+
 shared_ptr<VTuneDomain> domainptr;
```

Comment:
we should not use this global, could you please let me know why?

Reply:
There was another issue on this variable, I set it global variable for tracing.
I removed the global definition in version 0.8.